core/app/c/[communitySlug]/settings/tokens/CreateTokenForm.tsx

```diff
@@ -61,6 +61,10 @@ export type CreateTokenForm = ReturnType<typeof useForm<CreateTokenFormSchema>>;
 export const CreateTokenForm = ({ context }: { context: CreateTokenFormContext }) => {
 	const form = useForm<CreateTokenFormSchema>({
 		resolver: zodResolver(createTokenFormSchema),
+		defaultValues: {
+			// default to 1 day from now, mostly to make testing easier
+			expiration: new Date(Date.now() + 1000 * 60 * 60 * 24),
+		},
```

Comment:
i didn't want to write the code to navigate the date picker, so i took the easy way out and set a default time instead. let me know if the default time should be longer or if we should do this differently.